modules/installation-azure-marketplace-subscribe.adoc

```diff
@@ -129,7 +129,39 @@ ifdef::ipi[]
 . Record the image details of your offer. You must update the `compute` section in the `install-config.yaml` file with values for `publisher`, `offer`, `sku`, and `version` before deploying the cluster.
 endif::ipi[]
 ifdef::upi[]
-. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes, you can update `storageProfile.imageReference` by deleting the `id` parameter and adding the `offer`, `publisher`, `sku`, and `version` parameters using the values from your offer.
+. Record the image details of your offer. If you use the Azure Resource Manager (ARM) template to deploy your worker nodes:
```

Comment:
What are the steps if you don't use the Azure Resource Manager template to deploy your worker nodes?

Reply:
The doc update request targets a specific update. Covering other routes is out of scope.